<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserCutOutputPage</class>
 <widget class="QWizardPage" name="LaserCutOutputPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxBottomStencil">
     <property name="text">
      <string>Bottom stencil</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPathEdit" name="patheditBottomStencil">
     <property name="pathMode">
      <enum>QPathEdit::AnyFile</enum>
     </property>
     <property name="nameFilters">
      <stringlist>
       <string>SVG files (*.svg)</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPathEdit" name="patheditTopStencil">
     <property name="pathMode">
      <enum>QPathEdit::AnyFile</enum>
     </property>
     <property name="nameFilters">
      <stringlist>
       <string>SVG files (*.svg)</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxTopStencil">
     <property name="text">
      <string>Top stencil</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBoxMirrorTop">
     <property name="text">
      <string>Mirror</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="checkBoxMirrorBottom">
     <property name="text">
      <string>Mirror</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPathEdit</class>
   <extends>QWidget</extends>
   <header>qpathedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
